auth/downscope/downscope.go

```diff
@@ -28,12 +28,11 @@ import (
 
 var identityBindingEndpoint = "https://sts.googleapis.com/v1/token"
 
-// Options for configuring [NewTokenProvider].
+// Options for configuring [NewCredentials].
 type Options struct {
-	// BaseProvider is the [cloud.google.com/go/auth.TokenProvider] used to
-	// create the downscoped provider. The downscoped provider therefore has
-	// some subset of the accesses of the original BaseProvider. Required.
-	BaseProvider auth.TokenProvider
+	// Credentials is the [cloud.google.com/go/auth.Credentials] used to
+	// create the downscoped provider. Required.
```

Comment:
Rename "provider" to "credentials" throughout these docs, where the term refers to the return value of `NewCredentials`.